backend/app.js

```diff
@@ -0,0 +1,34 @@
+const { connect, Account, keyStores: { InMemoryKeyStore } } = require('near-api-js');
+
+(async function () {
+    const config = require('./config')(process.env.NODE_ENV || 'development')
+    // TODO: Why no default keyStore?
+    const keyStore = new InMemoryKeyStore();
+    const near = await connect({...config, keyStore});
+    const account = await near.account('berryclub.ek.near');
+    const pixelsState = await account.viewState('p');
```

Comment:
@evgenykuzyakov is there any way to call https://docs.near.org/docs/api/rpc#view-contract-state with specific block index?

Reply:
This code uses https://github.com/near/near-api-js/pull/452

Reply:
yeah you can pass `block_id` in the query

Reply:
In the RPC params, change `"finality": "..."` to `"block_id": <BLOCK_HEIGHT>`